pkg/services/sso/redhatsso_service.go

```diff
@@ -26,11 +26,12 @@ func (r *redhatssoService) RegisterClientInSSO(accessToken string, clusterId str
 }
 
 func (r *redhatssoService) DeRegisterClientInSSO(accessToken string, clientId string) *errors.ServiceError {
+	glog.V(5).Infof("Deregistering client with id: %s in redhat_sso", clientId)
```

Comment:
This log is not adding any custom info for redhat_sso. It can be moved to the proxy:
```
glog.V(5).Infof("Deregistering client with id: %s in %s", clientId, r.GetConfig().SelectSSOProvider)
```